chromium_src/ui/base/resource/resource_bundle.cc

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "ui/base/resource/resource_bundle.h"
+
+#include "base/containers/flat_set.h"
+#include "base/no_destructor.h"
+
+#define GetThemedLottieImageNamed GetThemedLottieImageNamed_ChromiumImpl
+#include <ui/base/resource/resource_bundle.cc>
+#undef GetThemedLottieImageNamed
+
+namespace ui {
+
+// Runtime set of blocked resource IDs
+static base::NoDestructor<base::flat_set<int>> g_blocked_lottie_ids;
+
+COMPONENT_EXPORT(UI_BASE)
+void SetBlockedThemedLottieImages(const base::flat_set<int>& blocked_ids) {
+  *g_blocked_lottie_ids = blocked_ids;
+}
+
+const ui::ImageModel& ResourceBundle::GetThemedLottieImageNamed(
+    int resource_id) {
+  // Don't display themed assets that are on our block list
+  if (g_blocked_lottie_ids->count(resource_id)) {
```

Comment:
talking to @rebron we want to block all of these, but returning an empty image will break in some places so we could override `GetLottieData` to return a hardcoded json value that corresponds to an empty image in lottie so it returns a valid value, but displays nothing?

Reply:
I think this should work 
```
{
  "v": "5.5.7",
  "fr": 1,
  "ip": 0,
  "op": 0,
  "w": 0,
  "h": 0,
  "nm": "Blank",
  "ddd": 0,
  "assets": [],
  "layers": []
}
```

Reply:
Yeah, I was trying something similar, but unfortunately zero-sized canvases are prohibited:

```
[63272:64112:0909/173656.887:ERROR:cc\paint\skottie_wrapper_impl.cc:46] [Skottie] "Invalid animation params (size: [0.000000 0.000000], frame rate: 30.000000, in-point: 0.000000, out-point: 1.000000)"
```

If I use a 1x1 canvas instead, it does the right thing for two of the images I'm trying to hide. However, the image we want to hide on the password bubble seems to get inflated up to some "preferred size", so while it's blank it also takes up a ton of layout space.

Reply:
OK, updated to block all Lottie assets.